src/_gettsim_tests/test_data/arbeitsl_geld_2/2005/hh_id_8.yaml

```diff
@@ -66,9 +66,9 @@ inputs:
       - 0.0
     jahr:
       - '2005'
-    wohngeld_vor_vermög_check_m_wthh:
+    wohngeld_anspruchshöhe_m_bg:
       - 0.0
-    wohngeld_vor_vermög_check_m_bg:
+    wohngeld_basisbetrag_m_bg:
```

Comment:
I think this is gone and can be deleted in all tests?

Reply:
Oh yes, no idea how that survived.